# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pcc/multilibdir.patch
# Copyright (C) 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- pcc-1.1.0/cc/cc/cc.c.vanilla	2019-04-21 11:23:24.058810440 +0000
+++ pcc-1.1.0/cc/cc/cc.c	2019-04-21 11:23:47.930810341 +0000
@@ -206,7 +206,7 @@
 #define PCCLIBDIR	NULL
 #endif
 #ifndef DEFLIBDIRS	/* default library search paths */
-#ifdef MULTIARCH_PATH
+#ifndef MULTIARCH_PATH
 #define DEFLIBDIRS	{ "/usr/lib/", 0 }
 #else
 #define DEFLIBDIRS	{ "/usr/lib/", "/usr/lib/" MULTIARCH_PATH "/", 0 }
